SPECS/vim/vim.spec

```diff
@@ -1,25 +1,25 @@
 %define debug_package %{nil}
 Summary:        Text editor
 Name:           vim
-Version:        8.1.1667
-Release:        2%{?dist}
+Version:        8.2.3582
+Release:        1%{?dist}
 License:        Vim
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/Editors
 URL:            https://www.vim.org
-#Source0:       https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz
-Source0:        %{name}-%{version}.tar.gz
+Source0:       https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
Nit-pick: could you aling the spaces with the other tags? Running a linter should fix all such issues.